### PR TITLE
Deploying 2021.03 version to covid19 Prod 1617317929

### PR DESCRIPTION
**Pull request by themarcelor**
copying files from qa-covid19.planx-pla.net to chicagoland.pandemicresponsecommons.org

### CODE REVIEW

File: chicagoland.pandemicresponsecommons.org/etlMapping.yaml [hunk 1]
```diff
@@ -12,6 +12,10 @@ mappings:
           - Korea, South: 'South Korea'
           - Taiwan*: 'Taiwan'
       - name: community_area
+        value_mappings:
+          - GREATER GRAND CROSSING: 'Grand Crossing'
+          - SOUTH LAWNDALE (LITTLE VILLAGE): 'Little Village'
+          - WEST ENGLEWOOD: 'W.Englewood'
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/etlMapping.yaml:18]: we're not doing this in prod

File: chicagoland.pandemicresponsecommons.org/manifest.json [hunk 2]
```diff
@@ -41,10 +43,10 @@
     "logs_bucket": "s3logs-logs-covid19-prod-gen3",
     "sync_from_dbgap": "False",
     "dispatcher_job_num": "10",
-    "netpolicy": "on",
+    "netpolicy": "off",
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/manifest.json:46]: `on`

File: chicagoland.pandemicresponsecommons.org/manifest.json [hunk 3]
```diff
@@ -212,28 +152,193 @@
...
+        "index": "chicagoland.pandemicresponsecommons.org_dataset",
         "type": "dataset"
       }
     ],
-    "config_index": "covid19_array-config",
+    "config_index": "chicagoland.pandemicresponsecommons.org_array-config",
     "auth_filter_field": "auth_resource_path",
     "aggs_include_missing_data": false
+  },
+  "mariner": {
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/manifest.json:179]: we're not deploying this to prod

File: chicagoland.pandemicresponsecommons.org/manifests/hatchery/hatchery.json [hunk 4]
```diff
@@ -5,20 +5,35 @@
...
     {
       "target-port": 8787,
       "cpu-limit": "1.0",
       "memory-limit": "1024Mi",
-      "name": "R Studio-3.5",
+      "name": "R Studio 3.5",
       "image": "quay.io/cdis/rstudio:master",
-      "env": {"DISABLE_AUTH": "true"},
+      "env": {
+        "DISABLE_AUTH": "true"
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/manifests/hatchery/hatchery.json:35]: anyone knows what this does?

File: chicagoland.pandemicresponsecommons.org/manifests/hatchery/hatchery.json [hunk 5]
```diff
@@ -55,6 +72,67 @@
       "ready-probe": "/",
       "user-volume-location": "/home/rstudio/pd",
       "fs-gid": 100
+    },
+    {
+      "target-port": 5800,
+      "cpu-limit": "1.0",
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/manifests/hatchery/hatchery.json:78]: this one is not ready for prod

File: chicagoland.pandemicresponsecommons.org/manifests/hatchery/jupyter-covid19.yaml [hunk 6]
```diff
@@ -6,9 +6,9 @@ services:
          - ${DATA_VOLUME}:/data
          - ${USER_VOLUME}:/home/jovyan/pd
       environment:
-        - NAMESPACE=default
-        - HOSTNAME=chicagoland.pandemicresponsecommons.org
-        - FRAME_ANCESTORS=https://chicagoland.pandemicresponsecommons.org
+        - NAMESPACE=qa-covid19
+        - HOSTNAME=qa-covid19.planx-pla.net
+        - FRAME_ANCESTORS=https://qa-covid19.planx-pla.net
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/manifests/hatchery/jupyter-covid19.yaml:11]: uh oh, this should be added to environment specific fields

File: chicagoland.pandemicresponsecommons.org/manifests/hatchery/jupyter-covid19.yaml [hunk 7]
```diff
@@ -26,8 +26,8 @@ services:
       deploy:
         resources:
           limits:
-            cpus: '4.0'
-            memory: 16Gi
+            cpus: '1.00'
+            memory: 1024M
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/manifests/hatchery/jupyter-covid19.yaml:30]: to reverse

File: chicagoland.pandemicresponsecommons.org/manifests/scaling/scaling.json [hunk 8]
```diff
@@ -0,0 +1,57 @@
+{
+  "arborist": {
+    "min": 1,
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/manifests/scaling/scaling.json:3]: these are not the right values, see the current manifest

File: chicagoland.pandemicresponsecommons.org/portal/gitops.json [hunk 9]
```diff
@@ -151,7 +151,6 @@
             "fields": [
               "date_collected",
               "investigator_affiliation",
-              "name",
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/portal/gitops.json:154]: we should revert this

File: chicagoland.pandemicresponsecommons.org/portal/gitops.json [hunk 10]
```diff
@@ -351,12 +352,13 @@
...
           },
-          { "field": "pO2_saturation",
+          {
+            "field": "pO2_saturation",
             "name": "pO2 Saturation (percent)"
           },
           {
             "field": "leukocyte_count",
-            "name": "Leukocyte Count (10^3/µl)"
+            "name": "Leukocyte Count (10^3/\u00b5l)"
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/portal/gitops.json:361]: probably an encoding issue in the gen3release-sdk tool?

File: chicagoland.pandemicresponsecommons.org/portal/gitops.json [hunk 11]
```diff
@@ -951,13 +961,13 @@
           },
           {
             "field": "hhinc_mean",
-            "name": "Mean Household Income (USD)"
+            "name": "Mean Household Income (US Currency"
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/portal/gitops.json:964]: need a closing parenthesis...

File: chicagoland.pandemicresponsecommons.org/portal/gitops.json [hunk 12]
```diff
@@ -1008,7 +1020,7 @@
         "imageUrl": "/dashboard/Public/notebooks/COVID-19-JHU_data_analysis_04072020.png"
       },
       {
-        "title": "A Live View of COVID-19’s Global Presence",
+        "title": "A Live View of COVID-19\u2019s Global Presence",
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/portal/gitops.json:1023]: encoding issue

File: chicagoland.pandemicresponsecommons.org/portal/gitops.json [hunk 13]
```diff
@@ -1062,15 +1074,8 @@
...
-  "componentToResourceMapping": {
-    "Workspace": {
-      "resource": "/workspace",
-      "method": "access",
-      "service": "jupyterhub"
-    }
-  },
   "covid19DashboardConfig": {
-    "dataUrl": "https://opendata.datacommons.io/",
+    "dataUrl": "https://static.planx-pla.net/",
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/portal/gitops.json:1078]: put this back

File: chicagoland.pandemicresponsecommons.org/portal/gitops.json [hunk 14]
```diff
@@ -1135,6 +1140,30 @@
                 ]
               }
             }
+          },
+          {
+            "title": "Total Deaths per 10,000 in Chicago",
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/portal/gitops.json:1145]: not ready for prod

File: chicagoland.pandemicresponsecommons.org/portal/gitops.json (hunk 11 above)
Comment (themarcelor) [reply chicagoland.pandemicresponsecommons.org/portal/gitops.json:964]: Czech this out: https://github.com/uc-cdis/gitops-qa/blob/master/qa-covid19.planx-pla.net/portal/gitops.json#L954

File: chicagoland.pandemicresponsecommons.org/portal/gitops.json (hunk 13 above)
Comment (themarcelor) [reply chicagoland.pandemicresponsecommons.org/portal/gitops.json:1078]: I need to tweak the automation config to skip that environment-specific setting.

File: chicagoland.pandemicresponsecommons.org/manifests/hatchery/jupyter-covid19.yaml (hunk 6 above)
Comment (themarcelor) [reply chicagoland.pandemicresponsecommons.org/manifests/hatchery/jupyter-covid19.yaml:11]: Correct. Let me create a PR..

File: chicagoland.pandemicresponsecommons.org/manifests/scaling/scaling.json (hunk 8 above)
Comment (themarcelor) [reply chicagoland.pandemicresponsecommons.org/manifests/scaling/scaling.json:3]: This is being copied from qa-covid19... 🤷🏼‍♂️ 
We can suppress this copy though.

File: chicagoland.pandemicresponsecommons.org/manifests/hatchery/hatchery.json (hunk 4 above)
Comment (mfshao) [reply chicagoland.pandemicresponsecommons.org/manifests/hatchery/hatchery.json:35]: i think this is bypassing the login page of the rstudio container

File: chicagoland.pandemicresponsecommons.org/portal/gitops.json (hunk 11 above)
Comment (paulineribeyre) [reply chicagoland.pandemicresponsecommons.org/portal/gitops.json:964]: yeah i know :p

**Comment by atharvar28** [comment]
Closing this PR as [#2733](https://github.com/uc-cdis/cdis-manifest/pull/2733) was merged with 2021.03 versions

**Pull request closed without merging**